index.ts

```diff
@@ -186,19 +194,119 @@ const llamaInvoke = (prompt: string, input: string): Promise<string> => {
     });
   });
 }
+
+const llamaInvokeWithAudio = async (prompt: string, input: string): Promise<string> => {
+  const formattedPrompt = `### Instruction:\n ${prompt} \n ### Input:\n ${input} \n ### Response:\nagent: `;
+  return new Promise((resolve, reject) => {
+    const promptTokens: string[] = [];
+    let promptTokensDoneEchoing: boolean = false;
+    let sentenceEndRegex = /[.!?,;]/;  // Adjust as necessary.
+    let promisesChain = Promise.resolve();
```

Comment:
node.js is the best programming language in the world

Reply:
i cannot be trusted with an event loop

Reply:
I'm still in awe you chose to write this in Typescript rather a Python, a language which would perfectly integrate with most ML loops, for a failed child of microsoft

Reply:
I wanted an asynchronous event loop that Just Worked
Any inference code should be compiled
Mark my words
Most inference ML will be compiled C called from javascript